diem-move/transaction-replay/src/lib.rs

```diff
@@ -125,12 +125,16 @@ impl DiemDebugger {
         payload: &WriteSetPayload,
         save_write_set: bool,
     ) -> Result<TransactionOutput> {
-        let state_view = DebuggerStateView::new(&*self.debugger, version + 1);
+        ensure!(
+            version > 0,
+            "Can't run a write set transaction without root account."
+        );
+        let state_view = DebuggerStateView::new(&*self.debugger, version.checked_sub(1));
         let vm = DiemVM::new(&state_view);
         let cache = diem_vm::data_cache::StateViewCache::new(&state_view);
         let sequence_number = match self
             .debugger
-            .get_account_state_by_version(diem_root_address(), version)?
+            .get_account_state_by_version(diem_root_address(), version - 1)?
```

Comment:
nit: if we're doing `checked_sub` in other places (e.g., above) why not introduce a new variable and reuse it?
```let execute_version = version.checked_sub(1)?;```

Reply:
Well, you need to use `.ok_or_else()?` to ensure a `Some`. Lmk if you like the fixup.